components/lists/payments/controller.tsx

```diff
@@ -0,0 +1,72 @@
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+
+import PaymentsListView from "components/lists/payments/view";
+import { TotalFiatNetworks } from "components/profile/pages/payments";
+
+import { Payment } from "interfaces/payments";
+
+import { NetworkPaymentsData } from "types/api";
+
+import { useNetwork } from "x-hooks/use-network";
+
+interface PaymentsListProps {
+  payments: NetworkPaymentsData[];
+  totalNetworks: TotalFiatNetworks[];
+  symbol: string;
+}
+
+// TODO: Add InfiniteScroll and pagination
```

Comment:
@moshmage Not yet, and I think this could be done later, because isn't expected for this list to have many items to soon, and this feature is taking more time then it was planned.